src/YoutubeAtomPlayer.tsx

```diff
@@ -306,6 +306,14 @@ const createInstantiateImaManager =
 
             const onAdsManagerLoaded = () => {
                 adsManager.current = imaManager.current?.getAdsManager();
+                const adsRenderingsettings =
```

Comment:
Do we need to wait to call `updateAdsRenderingSettings` until after the ads manager has loaded? Or could this go directly after instantiating the `imaManager` above?

Reply:
Once `adsRenderingsettings` is created we set it with `adsManager.current?.updateAdsRenderingSettings(adsRenderingsettings);`
So yes you're right we could create it earlier but I thought it would be a bit tidier to keep it closer to where it was being used.